migrations/tests/test_convert_cache_to_dataclass_v1.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""Unit tests for convert_cache_to_dataclass_v1.py."""
+import io
+import os
+import pickle
+import sys
+import tempfile
+from unittest.mock import patch
+
+import pytest
+
+import migrations.convert_cache_to_dataclass_v1 as migration  # noqa:I202
+
+
+@pytest.fixture
+def fixture_bak_cleanup():
+    """Cleanup bak file which is created during migration."""
+    bak_fnames = []
+
+    def _fixture_back_cleanup(bak_fname):
+        bak_fnames.append(bak_fname)
+        return bak_fname
+
+    yield _fixture_back_cleanup
+
+    for bak_fname in bak_fnames:
+        print("teardown of '{}'".format(bak_fname))
+        os.remove(bak_fname)
+
+
+@pytest.fixture
+def fixture_cache_file():
```

Comment:
This is due to the fact that `conftest.py` would have to be in the root which I dislike and I don't know how to solve it at the moment.